src/main.rs

```diff
@@ -207,10 +206,10 @@ fn show_config(config: &config::Config) {
             .clone()
             .map(|t| t.name.unwrap_or("none".to_string()))
             .unwrap_or("none".to_string()),
-        width = config
-            .width
-            .clone()
-            .unwrap_or((Term::stdout().size().1 - 1).to_string()),
+        width = match config.decorations_width {
+            cli::Width::Fixed(width) => width.to_string(),
+            _ => config.available_terminal_width.to_string()
+        },
```

Comment:
I thought `fn show_config` should show `available_terminal_width` if user doesn't assign any number to width, but if you think it is not as intended, please tell me.

Reply:
Right, I do think that if the user does e.g. `delta --width variable --show-config` then we need it to print out `width = variable` here, so that the output faithfully represents the active config values.

Reply:
fixed here
4284d6e
<img width="615" alt="ss 5" src="https://user-images.githubusercontent.com/41639488/88976033-49ca3d80-d2f6-11ea-8748-415b09303a61.png">